### PR TITLE
feature Parasite Hub

### PR DESCRIPTION
**Pull request by ariannapa**
Python function

### CODE REVIEW

**Review by EttoreSaetta — CHANGES_REQUESTED** [review]
You removed the other functions.
Fix it!

**Comment by ariannapa** [comment]
﻿
﻿
Buongiorno, posso risolvere rimettendo le funzioni nella cartella dedicata, giusto? Avevo citato questo problema nell’email  di martedì.
Cordiali saluti.

Il giorno 22 lug 2024, alle ore 10:12, SaettaE ***@***.***> ha scritto:

﻿

@EttoreSaetta requested changes on this pull request.

You removed the other functions.
Fix it!

File: Helicopter_Class.py [hunk 1]
```diff
@@ -12,13 +10,4 @@ def __init__(self, Omega_r_mr, R_mr, R_hub, c_mr, N_mr, theta0, theta_tw, Cla, C
...
-        # Tail rotor
-        self.Omega_r_tr = Omega_r_tr         # Tail rotor tip speed (m/s)
-        self.R_tr = R_tr                     # Tail rotor radius (m)
-        self.R_hub_tr = R_hub_tr             # Tail Rotor hub radius (m)
-        self.c_tr = c_tr                     # Tail rotor chord (m)
-        self.N_tr = N_tr                     # Number of blades of tail rotor
-        self.l_tr = l_tr                     # Distance between tail rotor and main rotor
-
-
         # Developing the possibility of integration within the feature_geometry
```
Comment (EttoreSaetta) [inline Helicopter_Class.py:13]: Are you sure these should be removed?

**Review by EttoreSaetta — CHANGES_REQUESTED** [review]

File: parasite_hub.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+#Academic Year 2023-2024
+#Date:14/07/2024
+#
+#Version:1.0.0
+
+
+
+import math as ma
+
+    def hub(self):
```
Comment (EttoreSaetta) [inline parasite_hub.py]: Remove indentation.
Self is used within the classes, rename your input.

File: parasite_hub.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        '''
+        This function calculates the parasite drag area of the hub, assuming it as a circular section
+        cylinder.
+        
+        Input:
+        -R:hub radius
+        -Lcs: hub height
+        -alpha: angle of attack
+        -S: reference area
```
Comment (EttoreSaetta) [inline parasite_hub.py]: Seems the input is wrong.
It looks like you only have 1 input variable you named "self".

**Review by EttoreSaetta — CHANGES_REQUESTED** [review]
Changes requested!

Also need a test case for your function.

**Review by EttoreSaetta — APPROVED** [review]
LGTM

**Pull request merged**